.changeset/eighty-gifts-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/ssr': patch
+---
+
+Fix adding node marker for hydration for nested custom elements without attributes.
```

Comment:
Nit: This contains *what* the change is but not the *why*. Is it worth mentioning the why?

I quite like the comment in the test: https://github.com/lit/lit/pull/3942/files#diff-e5374f241814044cc1f766670d048d5cb724a230333deedc677dfb47ac606b50R5267-R5269

Reply:
Good suggestion. Done!